src/main/java/li/cil/oc2/common/bus/device/DeviceTypes.java

```diff
@@ -31,7 +33,8 @@ public static void initialize() {
 
         DEVICE_TYPES.register(FMLJavaModLoadingContext.get().getModEventBus());
 
-        FMLJavaModLoadingContext.get().getModEventBus().addListener(DeviceTypes::handleTextureStitchEvent);
```

Comment:
Ah, the event type references a client class, I see.

Could you find out if it's too late to register this in `ClientSetup`?

Reply:
Yea i moved it to ClientSetup